src/test/java/com/iota/iri/service/tipselection/impl/CumulativeWeightCalculatorTest.java

```diff
@@ -18,7 +18,6 @@
 import org.junit.AfterClass;
 import org.junit.Assert;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
```

Comment:
So we're still using transforming map
You want to avoid doing more tests for now?

Fine with that, we can change it later